spirv_msl.cpp

```diff
@@ -9339,7 +9607,12 @@ void CompilerMSL::entry_point_args_builtin(string &ep_args)
 			     bi_type != BuiltInViewIndex) &&
 			    (get_execution_model() == ExecutionModelGLCompute ||
 			     (get_execution_model() == ExecutionModelFragment && msl_options.supports_msl_version(2, 2)) ||
-			     (bi_type != BuiltInSubgroupLocalInvocationId && bi_type != BuiltInSubgroupSize)))
+			     (bi_type != BuiltInSubgroupLocalInvocationId && bi_type != BuiltInSubgroupSize)) &&
```

Comment:
This branch has to be setting some kind of record for unreadability at this point ...

Reply:
I agree. So I refactored it in #1430.